Resources/public/js/views/services/ez-dashboardblocksviewservice.js

```diff
@@ -20,5 +23,163 @@ YUI.add('ez-dashboardblocksviewservice', function (Y) {
      * @constructor
      * @extends eZ.ViewService
      */
-    Y.eZ.DashboardBlocksViewService = Y.Base.create('dashboardBlocksViewService', Y.eZ.ViewService, [], {});
+    Y.eZ.DashboardBlocksViewService = Y.Base.create('dashboardBlocksViewService', Y.eZ.ViewService, [], {
+        _load: function (next) {
+            this._loadRootLocation()
+                .then(Y.bind(this._loadRootLocationModelData, this))
+                .then(next)
+                .catch(Y.bind(this._handleError, this));
```

Comment:
We should also be able to provide more details about the error. The error message might be not enough to debug an issue.